docs/using/installing/index.rst

```diff
@@ -4,17 +4,21 @@
 Installing Flocker
 ==================
 
-As a user of Flocker you will need to install the ``flocker-cli`` package which provides command line tools to control the cluster.
-This should be installed on a machine with SSH credentials to control the cluster nodes
-(e.g., if you use our Vagrant setup then the machine which is running Vagrant).
+The Flocker CLI is installed on your local machine via the ``flocker-cli`` package which provides command line tools to control the cluster. 
+This package also includes the ``flocker-ca`` tool, which you must use to generate certificates for all the Flocker components.
 
-There is also a ``clusterhq-flocker-node`` package which is installed on each node in the cluster.
-It contains the services that need to run on each node.
+The Flocker agents are installed on any number of nodes (or hosts) in the cluster where your containers will run.
```

Comment:
This clarification was suggested by @jongiddy at https://github.com/ClusterHQ/flocker/pull/1477#discussion_r31351034 as a workaround to the image which has now been removed. Stick with hosts or nodes instead of both. I prefer nodes.